python/lsst/ts/GenericCamera/driver/simulatorcamera.py

```diff
@@ -430,3 +435,26 @@ async def readout(self):
 
         # Reset readout state
         self.readout_state = 0
+
+    async def _set_tag_values(self):
+        """Convenience coroutine to provide values for some of the tags in the
+        FITS header. More tags can be added if necessary but these were deemed
+        sufficient for unit testing."""
+        # Add the Canon-related FITS header items to the generic ones.
+        fhig = FitsHeaderItemsGenerator()
```

Comment:
Same here. either make it in one line or improve name.

Reply:
I made it one line.